.circleci/config.yml

```diff
@@ -93,14 +93,11 @@ jobs:
       - attach_workspace:
           at: ~/ld-react-components
       - run: git config user.email dave@cmscode.com
-      - run: git config user.name "poorpaddy"  
+      - run: git config user.name "poorpaddy"
       - run: mkdir ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
       - run:
           name: Build Docs
-          command: yarn docs:build
-      - run:
-          name: Publish Docs
-          command: yarn docs:publish
+          command: yarn deploy:storybook
```

Comment:
This points to `"deploy:storybook": "storybook-to-ghpages"` in package.json. What does `storybook-to-ghpages` actually do? I see no other reference to it? We also have to ensure this actual push to gh-pages branch contains a message with `[skip ci]` or CircleCi will try to build the gh-pages branch and fail because of the lack of config.yml.